cmd/blockchaincmd/deploy.go

```diff
@@ -599,13 +599,18 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
 				avagoVersionSettings := node.AvalancheGoVersionSettings{}
 				useEtnaDevnet := network.Kind == models.EtnaDevnet
 				if avagoBinaryPath == "" {
-					ux.Logger.PrintToUser("Local build of Avalanche Go is required to create an Avalanche node using local machine")
-					ux.Logger.PrintToUser("Please download Avalanche Go repo at https://github.com/ava-labs/avalanchego and build from source through ./scripts/build.sh")
-					ux.Logger.PrintToUser("Please provide the full path to Avalanche Go binary in the build directory (e.g, xxx/build/avalanchego)")
-					avagoBinaryPath, err = app.Prompt.CaptureString("Path to Avalanche Go build")
+					avaGoVersionSetting := node.AvalancheGoVersionSettings{
+						UseLatestAvalanchegoPreReleaseVersion: true,
```

Comment:
do we need to include `userProvidedAvagoVersion` into the `node.AvalancheGoVersionSettings`?